### DISCUSSION TITLE
How do you uninstall all the files auto config created?

### DISCUSSION THREAD

**Discussion by katxl** [ptrumpis/snap-camera-server#26] [Q&A] [answered]
I made a mistake with not giving webapp-1 permission, and I can't figure out how to give it, maybe I could uninstall everything and start over? How do I get rid of the files auto config created and reinstall?

**Comment by ptrumpis** [comment, marked as answer]
> How do you uninstall all the files auto config created?

You have to manually undo the installation steps described here: [Configuration](https://github.com/ptrumpis/snap-camera-server/blob/main/docs/CONFIGURATION.md)
Usually you don't have to undo them. You can run it multiple times and it should overwrite previous settings.

> I made a mistake with not giving webapp-1 permission, and I can't figure out how to give it, maybe I could uninstall everything and start over?

What permission are you talking about?
I assume you are receiving an error message right now.
Please share the original error message.

**Reply by katxl** [reply]
Hi, I deleted and reinstalled Docker, snap camera works now. Thanks!
